samples/confidential_ml/Makefile

```diff
@@ -13,12 +9,12 @@ SHUTDOWN_WAIT = 2
 all: $(PACKAGE)
 
 appdir:
-	$(APPBUILDER) -m -d Dockerfile
+	@myst-appbuilder -m -d Dockerfile
 
 $(PACKAGE): appdir package.pem
-	$(MYST) mkext2 appdir rootfs
-	$(MYST) fssig --roothash rootfs > roothash
-	$(MYST) package-sgx --roothash=roothash package.pem config.json
+	@myst mkext2 appdir rootfs
```

Comment:
I'd prefer having it without the @, that way we can see what commands were run in the pipeline logs. For end-user it is not a big difference in experience.